ddspipe_participants/src/cpp/participant/dds/XmlParticipant.cpp

```diff
@@ -99,7 +96,7 @@ fastdds::dds::DomainParticipantQos XmlParticipant::reckon_participant_qos_() con
             qos
             );
 
-        if (res != fastrtps::types::ReturnCode_t::ReturnCodeValue::RETCODE_OK)
+        if (res != fastdds::dds::RETCODE_OK)
```

Comment:
I've just realized "common" properties are lost if a profile is set, we should change that. For example, by having a new `add_qos_properties` virtual method, with qos passed by reference.